pages/my-account.vue

```diff
@@ -159,7 +245,112 @@ export default defineComponent({
   }
 }
 
-.breadcrumbs {
-  padding: var(--spacer-base) 0 var(--spacer-lg);
+div.kibo-sf-accordion-item {
+  border: none;
+  border-bottom: 1px solid var(--_c-white-secondary);
+}
+
+.sf-accordion-item {
+  @include for-mobile {
+    padding: 0 calc(var(--spacer-2xs) * 3.5);
+  }
+
+  ::v-deep &__content {
+    padding-top: 0;
+    border: none;
+  }
+
+  ::v-deep &__header {
+    justify-content: space-between;
+    color: var(--_c-dark-primary);
+    font-family: var(--font-family--primary);
+    font-size: var(--font-size--base);
+    line-height: calc(var(--spacer-sm) + var(--spacer-2xs));
+    text-align: left;
+    border: none;
+    height: calc(var(--spacer-xl) + var(--spacer-sm));
+  }
+}
+
+.title-bar {
+  display: flex;
+  justify-content: flex-start;
+  background-color: var(--_c-light-secondary);
+  padding-left: 0;
+}
+
+.sf-bar {
+  @include for-mobile {
+    margin-left: calc(var(--spacer-base) + var(--spacer-2xs));
+  }
+
+  ::v-deep &__title {
+    padding-left: calc(var(--spacer-2xs) * 3.5);
+  }
+}
+
+.profile-img-container {
+  display: flex;
+  justify-content: center;
+
+  &__pic {
+    width: var(--spacer-xl);
+    border-radius: 50%;
+  }
+}
+
+.header-text-font {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  color: var(--_c-dark-primary);
+  font-family: var(--font-family--primary);
+  font-size: calc(var(--font-size--base) + (var(--font-size--xs) / 3));
+  line-height: var(--spacer-base);
+  text-align: left;
+  font-weight: bold;
+  height: calc(var(--spacer-2xs) * 14.5);
+
+  @include for-mobile {
+    margin-left: calc(var(--spacer-base) + var(--spacer-2xs));
+  }
+}
+
+.vertical-space {
+  background-color: var(--_c-white-secondary);
+  width: 100%;
+  height: calc(var(--spacer-2xs) * 4.75);
+}
+
+.spacer {
+  margin-top: calc(var(--spacer-2xs) * 2.5);
+
+  @include for-mobile {
+    margin-left: calc(var(--spacer-2xs) * 7);
+  }
+}
+
+.profile-logout {
+  cursor: pointer;
+}
+
+.order-header-border {
+  border: none;
+  border-bottom: 1px solid var(--_c-white-secondary);
+}
+
+.order-history {
+  margin-left: calc(var(--spacer-2xs) * 3.5);
+  margin-right: calc(var(--spacer-2xs) * 3.5);
+}
+
+.header-text-weight {
+  font-weight: bold;
+  font-size: calc(var(--font-size--base) + (var(--font-size--xs) / 3));
```

Comment:
Can't we use already provided font sizes ?

Reply:
Actually we can, I only considered the internal one. Thank you.